docs/api-table.js

```diff
@@ -1941,5 +1941,29 @@ export const iOS = {
         'If the changes support animation',
       ],
     ],
-  }
+  },
+  Badge: {
+    required: [
+    [
+      'text', 
+      'String', 
+      '', 
+      'The text to be displayed on the badge'
+    ],
+    [
+      'variant',
+      'Warp.Badge.Variant.info <br />Warp.Badge.Variant.success <br />Warp.Badge.Variant.warning <br />Warp.Badge.Variant.error <br />Warp.Badge.Variant.disabled <br />Warp.Badge.Variant.sponsored <br />Warp.Badge.Variant.neutral',
```

Comment:
I thought we would keep consistency regarding variant naming, too. In web we have info, **positive** (instead of success), warning, **negative** (instead of error), disabled, **price** and neutral (but we also don't have sponsored - should we? @magnuh  🤔 )